frontend/app.py

```diff
@@ -9,12 +10,16 @@
 app = Flask(__name__)
 
 backend_url = os.getenv('BACKEND_URL', 'http://localhost:5000')
-create_url = f'''{backend_url}/tasks'''
+task_url = f'''{backend_url}/tasks'''
 
 
 @app.route("/")
-def hello():
-    return render_template('index.html')
+def index():
+    r = requests.get(task_url)
```

Comment:
raise_for_status
https://www.geeksforgeeks.org/response-raise_for_status-python-requests/
https://tonari-it.com/python-response-status-code/